app/assets/javascripts/admin-conference.js

```diff
@@ -271,15 +272,38 @@ require(["lib/models", "lib/tools", "lib/owned", "knockout", "ko.sortable", "dat
             console.log("saveConference::");
             //check fields
             if( Array.isArray(self.conference().mFigs()) ){ self.conference().mFigs(0); }
-            if(self.conference().mAbsLeng() == null){ self.conference().mAbsLeng(500); }
-            else if(self.conference().mAbsLeng() == 0){
-                self.setError("danger", "Max. Abs. Len. has to be larger than 0.")
+            if( self.conference().mAbsLeng() == null ){ self.conference().mAbsLeng(500); }
```

Comment:
A) Please use a space before the bracket, but not within the brackets whenever using an `if` clause.
B) Please do not put the body of an if in the same line.
In this case:
```
if (self.conference().mAbsLeng() == null) {
    self.conference().mAbsLeng(500);
}
```

Please also change in all the changed lines below, but not in unchanged code, we will properly cleanup the code another time.